app/views/provider/request_password_resets/new.html.slim

```diff
@@ -0,0 +1,8 @@
+= javascript_pack_tag 'ResetPasswordPage'
+- authentication_providers = (@authentication_providers || []).map { |ap| {authorizeURL: ap.authorize_url, humanKind: ap.human_kind} }
+- flash_messages = (flash || []).map { |f| {type: f[0], message: f[1]}}
```

Comment:
Are we expecting any error in this page?

Reply:
hey @damianpm

yeah.. this page can have errors.